pkg/kubelet/stats/cri_stats_provider_test.go

```diff
@@ -385,17 +412,21 @@ func checkCRIRootfsStats(assert *assert.Assertions, actual statsapi.ContainerSta
 	assert.Equal(cs.WritableLayer.InodesUsed.Value, *actual.Rootfs.InodesUsed)
 }
 
-func checkCRILogsStats(assert *assert.Assertions, actual statsapi.ContainerStats, rootFsInfo *cadvisorapiv2.FsInfo) {
-	assert.Equal(rootFsInfo.Timestamp, actual.Logs.Time.Time)
-	assert.Equal(rootFsInfo.Available, *actual.Logs.AvailableBytes)
-	assert.Equal(rootFsInfo.Capacity, *actual.Logs.CapacityBytes)
-	assert.Equal(*rootFsInfo.InodesFree, *actual.Logs.InodesFree)
-	assert.Equal(*rootFsInfo.Inodes, *actual.Logs.Inodes)
-	assert.Nil(actual.Logs.UsedBytes)
-	assert.Nil(actual.Logs.InodesUsed)
+func checkCRILogsStats(assert *assert.Assertions, actual statsapi.ContainerStats) {
```

Comment:
Compare the value for `rootFsInfo` and  log stats. That's the point of the fake, right? :)

Reply:
done